### PR TITLE
Enable the web link

### PR DESCRIPTION
**Pull request by ashhhleyyy**
https://stats.nucleoid.xyz/ is done so its time :tada:

### CODE REVIEW

File: src/main/java/xyz/nucleoid/extras/integrations/game/StatisticsIntegration.java [hunk 1]
```diff
@@ -69,11 +70,10 @@ private void handleStatisticsBundle(GameSpace space, String namespace, GameStati
...
         this.sendBundle(body);
-        // TODO: Enable this when the web thing is done
-//        space.getPlayers().sendMessage(new TranslatableText("text.nucleoid_extras.statistics.web_url", gameId)
-//                .formatted(Formatting.GRAY, Formatting.ITALIC)
-//                .styled(style -> style.withClickEvent(new ClickEvent(ClickEvent.Action.OPEN_URL,
-//                        "https://nucleoid.xyz/game/#" + gameId))));
+        space.getPlayers().sendMessage(new TranslatableText("text.nucleoid_extras.statistics.web_url", gameId)
+                .formatted(Formatting.GRAY, Formatting.ITALIC)
+                .styled(style -> style.withClickEvent(new ClickEvent(ClickEvent.Action.OPEN_URL,
+                        "https://stats.nucleoid.xyz/games/" + gameId))));
```
Comment (Gegy) [inline src/main/java/xyz/nucleoid/extras/integrations/game/StatisticsIntegration.java:76]: It’s not very important- can we have the URL sent from the backend? 🙂

**Review by Gegy — COMMENTED** [review]
🎉

**Comment by Patbox** [comment]
Oops, too quick merge :tiny_potato:

File: src/main/java/xyz/nucleoid/extras/integrations/game/StatisticsIntegration.java (hunk 1 above)
Comment (ashhhleyyy) [reply src/main/java/xyz/nucleoid/extras/integrations/game/StatisticsIntegration.java:76]: Maybe, but at the minute the backend doesn't really have a way of sending a 'reply' to a message, so maybe when its refactored at some point

**Pull request merged**